source/includes/stackpath/_sites.md

```diff
@@ -961,3 +961,36 @@ Optional | &nbsp;
 `backupOrigin.username` <br/> *string* | Username to use when authenticating with the backup origin. 
 `backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
 `backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
+
+<!-------------------- RETRIEVE EDGESSL SETTINGS -------------------->
```

Comment:
Should this be under sites? I am about to create a PR and I was thinking or putting that in `_edgessl_settings`. WDYT?

Reply:
I initially created an `_edgessl_settings` file as well but my reason for putting it under sites is because the WAF, Origin, and CDN settings were all under sites and the EdgeSSL settings are a similar type of settings (scoped to the site). Although, I don't have a preference and think having its own section can work too.